commands/mining.go

```diff
@@ -50,9 +50,7 @@ var miningStartCmd = &cmds.Command{
 
 var miningStopCmd = &cmds.Command{
 	Run: func(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment) error {
-		if err := GetAPI(env).Mining().Stop(req.Context); err != nil {
```

Comment:
Mining().Stop() already was never returning anything but nil error, same as node.StopMining(), so no point in checking for a fake error.